### DISCUSSION TITLE
"File not found" error with Ranger when using Flatpak version

### DISCUSSION THREAD

**Discussion by olstach** [johnfactotum/foliate#1301] [General]
**Describe the bug**
When the Flatpak version of Foliate is set as the default app for epub files in Ranger, it displays a "Book cannot be opened: File not found" error. The snap version works fine. 

**To Reproduce**
1. Add the following line to ~/.config/ranger/rifle.conf:
ext epub, has com.github.johnfactotum.Foliate, X, flag f = com.github.johnfactotum.Foliate "$@"
2. Start Ranger in a new terminal session and try to open an .epub file 

**Expected behavior**
Foliate opens the selected file correctly.

**Version:**
 - Foliate version: 3.1.0
 - OS/Distribution and version: Pop!_OS 22.04
 - Desktop environment: GNOME 42.5
 - Installation method: Flatpak

**Comment by johnfactotum** [comment]
Not familiar with Ranger, but to open files with Flatpak, you need to use `--file-forwarding`. See https://github.com/johnfactotum/foliate/issues/1249.
